### PR TITLE
docs(integrate): apache 2.0

### PR DESCRIPTION
**Pull request by mffap**
### Definition of Ready

- [ ] I am happy with the code
- [ ] Short description of the feature/issue is added in the pr description
- [ ] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] My code has no repetitions
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

**Comment by mffap** [comment]
@fforootd @livio-a I've transferred the example code to the zitadel organization. This PR depends loosely on the repo zitadel/example-apache2

File: docs/docs/guides/integrate/tools/apache2.mdx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+![Redirect Settings](/img/guides/integrate/tools/apache-redirect_settings.png)
+
+You can find the url to your discovery endpoint under "URLs":
+![Discovery Endpoint](/img/guides/integrate/tools/apache-urls.png)
+
+## Configure Apache2
+
+### Configure mod_auth_openidc
+
+We use the module module `mod_auth_openidc` in this guide.
```
Comment (livio-a) [inline docs/docs/guides/integrate/tools/apache2.mdx:23]:
```suggestion
We use the module `mod_auth_openidc` in this guide.
```

File: docs/docs/guides/integrate/tools/apache2.mdx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+OIDCPKCEMethod S256
+```
+
+With the following parameters
+
+| Parameter | Description | Example value|
+|---|---|---|
+| OIDCProviderMetadataURL | Is the url to the discovery endpoint, which is typically located at {your-domain}/.well-known/openid-configuration| https://<your_domain>.zitadel.cloud/.well-known/openid-configuration |
+| OIDCClientID | Is the ID of the zitadel application. You can find it on the settings page of the application.  | 123456789123@apache_test |
+| OIDCRedirectURI | Users will be redirected to this page after successful login. If you are using localhost make sure to enable development mode in ZITADEL. | http://localhost:8080/secure/callback |
```
Comment (livio-a) [inline docs/docs/guides/integrate/tools/apache2.mdx:45]: do we need to mention that, it's not only localhost, but rather any non http*s* endpoint?

**Review by livio-a — CHANGES_REQUESTED** [review]

File: docs/docs/guides/integrate/tools/apache2.mdx (hunk 2 above)
Comment (mffap) [reply docs/docs/guides/integrate/tools/apache2.mdx]: updated the text.

**Review by fforootd — APPROVED** [review]
Looks good to me

**Comment by mffap** [comment]
> Looks good to me

I think Livio has to approve the requested changes.

**Review by livio-a — APPROVED** [review]

**Pull request merged**